package.json

```diff
@@ -31,8 +38,16 @@
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-preset-es2015": "^6.3.13",
     "browserify": "^13.0.1",
+    "eslint": "^4.6.1",
+    "eslint-config-prettier": "^2.4.0",
+    "eslint-plugin-prettier": "^2.2.0",
     "mocha": "^2.4.5",
-    "nock": "^9.0.14"
+    "nock": "^9.0.14",
+    "prettier": "^1.7.0"
+  },
+  "optionalDependencies": {
```

Comment:
These 2 packages are optionalPackages as they require node 4.2, and break CI runs on travis for node versions < 4.2 if not optional.